bridges/FacebookBridge.php

```diff
@@ -162,8 +162,7 @@ private function extractGroupName($html) {
 		$ogtitle = $html->find('meta[property="og:title"]', 0)
 			or returnServerError('Unable to find group title!');
 
-		return htmlspecialchars_decode($ogtitle->content, ENT_QUOTES);
-
+		return html_entity_decode($ogtitle->content);
```

Comment:
`ENT_QUOTES` should be included as well, to make sure single quotes are decoded as well (like before)